signer/src/stacks/api.rs

```diff
@@ -547,14 +572,57 @@ impl StacksClient {
 }
 
 impl StacksInteract for StacksClient {
-    async fn get_block(&self, block_id: StacksBlockId) -> Result<NakamotoBlock, Error> {
-        self.get_block(block_id).await
+    async fn get_account(&mut self, address: &StacksAddress) -> Result<AccountInfo, Error> {
+        let result = StacksClient::get_account(self, address).await;
+        match result {
+            Err(Error::StacksNodeRequest(_)) | Err(Error::StacksNodeResponse(_)) => {
```

Comment:
It's not clear that you want to match on all `Error::StacksNodeRequest` and all `Error::StacksNodeResponse` variants here. Instead you probably want to check the type of request error in `Error::StacksNodeRequest` and match on whether it is a timeout error, connect error, and maybe some other kind. Matching on all of it is probably too broad, so for now just match on timeout and connect errors.

Reply:
Done.